core/base/ftmTreePP/FTMTreePPUtils.h

```diff
@@ -0,0 +1,49 @@
+/// \ingroup base
+/// \class ttk::FTMTreePPUtils
+/// \author Mathieu Pont (mathieu.pont@lip6.fr)
+/// \date 2021.
+///
+/// Utils function for manipulating FTMTree class
+
+#ifndef _FTMTREEPPUTILS_H
+#define _FTMTREEPPUTILS_H
+
+#pragma once
+
+#include <FTMTree.h>
+#include <FTMTreePP.h>
+#include <FTMTreeUtils.h>
+
+#include <ttkUtils.h>
```

Comment:
to remove

Reply:
In the next commit: ttkUtils will be removed. But FTMTreePP and FTMTreeUtils are needed to have the FTMTreePP class and a function in FTMTreeUtils.